src/app/screen-snippet-handler.ts

```diff
@@ -52,6 +52,10 @@ class ScreenSnippet {
     if (isLinux) {
       this.captureUtil = '/usr/bin/gnome-screenshot';
     }
+
+    ipcMain.on('send-tracking-data-to-main', async (_event, eventData: { element: AnalyticsElements, type: ScreenSnippetActionTypes }) => {
```

Comment:
Can we change the event name here? It looks very technical. It can be something like `snippet-event`